osmaxx/excerptexport/urls.py

```diff
@@ -3,28 +3,24 @@
 from django.views.generic import TemplateView
 
 from osmaxx.excerptexport.views import (
-    list_downloads,
     download_file,
-    extraction_order_status,
```

Comment:
Can we delete the code for view `extraction_order_status` or is it used somewhere besides here (and its own tests)?